### PR TITLE
Add in a link to a Code of Conduct

### PR DESCRIPTION
**Pull request by quaid**
If the OIF Code of Conduct is the wrong one, what is the correct one?

This should be easily findable in the footer or a main menu, depending on information architecture of the site.

I am proposing this link in this location as my best guest of what CoC to use and where to put the link.

If there should be a different CoC or location for the link, feel free to use this merge request to and patch it for the correct info.

### CODE REVIEW

File: src/components/Layout/Footer.js [hunk 1]
```diff
@@ -63,6 +63,14 @@ export const Footer = () => (
       >
         Creator of Operate-First Logo
       </Text>
+        <Text
+        component={TextVariants.a}
+        href="//www.openstack.org/legal/community-code-of-conduct/"
+        target="top"
+        aria-label="Code of Conduct"
+      >
+        Creator of Operate-First Logo
```
Comment (tumido) [inline src/components/Layout/Footer.js]: Please fix the link text

**Review by tumido — CHANGES_REQUESTED** [review]

File: src/components/Layout/Footer.js (hunk 1 above)
Comment (durandom) [reply src/components/Layout/Footer.js]: just "Code of Conduct" ?

**Comment by durandom** [comment]
I have no idea which CoC to use 🤣 🤷‍♂️
If the Openstack one is a good one, just use that, sounds reasonable.
Is there one we could fork? Like you would fork a style-guide?

File: src/components/Layout/Footer.js (hunk 1 above)
Comment (quaid) [reply src/components/Layout/Footer.js]: Is there something unclear about the link text?

The link is to serve two purposes—show people we have a code of conduct (people look for it when they come to a project), and if you need to read or use the CoC, there it is. I _think_ the link text of "Code of Conduct" is conveys that, but perhaps there is a more clear way to accomplish that?

**Comment by quaid** [comment]
> I have no idea which CoC to use rofl man_shrugging
> If the Openstack one is a good one, just use that, sounds reasonable.
> Is there one we could fork? Like you would fork a style-guide?

Here is my reasoning:

* There is a clear association/relationship with this project and the openinfra.dev.
* Open Infra Labs uses the Openstack CoC.
* It looks to me like a good CoC, derived from CoCs with a good heritage.
* It is adopted across the entire Open Infra Foundation hosted projects.
* It's easier to adopt the CoC of sibling community than to create one from scratch.
* Adopting the OpenStack CoC keeps things compatible in common spaces.

That reasoning all said, another viewpoint is to look to the Kubernetes CoC instead.

https://kubernetes.io/community/code-of-conduct/

That one has a provenance going to the Contributor Covenant, which is a very popular starting point for CoCs. The k8s community one is actually derived from the CNCFs.

So, that all said, what is the governance in this community to make a decision around which code of conduct is The Code of Conduct? (I was just guessing, too.)

Finally, as for where it should be linked, I'm not sure the footer is the best place, but it's somewhere to put it for now (presuming the project is adopting one.)

File: src/components/Layout/Footer.js (hunk 1 above)
Comment (tumido) [reply src/components/Layout/Footer.js]: Sorry, I should have explained myself better here.
1. You're correctly setting the `aria-label` to `Code of Conduct`.
2. The visible link text is left as `Creator of Operate-First Logo`. And that's what appears on the site if you render it/start it:

image

**Review by tumido — CHANGES_REQUESTED** [review]

File: src/components/Layout/Footer.js (hunk 1 above)
Comment (quaid) [reply src/components/Layout/Footer.js]: Oh my, yes, thank you, I was indeed not parsing the markup correctly, I'll fix the patch.

**Review by tumido — COMMENTED** [review]
/lgtm

@durandom can you please approve if you agree with the code of conduct choice? I have really no expertise in this. In my view I'd like to keep us rather closer to CNCF than OpenStack simply due to the nature of the projects we colaborate with mostly, but I really don't understand the implications at all.

**Pull request merged**